src/api.ts

```
@@ -16,7 +16,13 @@ function get<T>(url: string): Promise<T> {
 
   return wait(300)
     .then(() => fetch(fullURL))
-    .then(res => res.json());
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(new Error(`${response.status}`));
```

I think it's better to throw the error here, not just reject